posts/pranithavallabhaneni/10-05-2022-pranithavallabhaneni-clicky-turtle-reflection.md

```diff
@@ -0,0 +1,23 @@
+---
+layout :  post
+author :  pranithavallabhaneni
+title  : "Pranitha's clicky turtle reflecttion "
+---
+For this clicky turtle hack my thoughts was to create a different trapeziums by click and to change back ground colors by using arrow keys. 
+
+**Difficulties:**
+First difficulty I faced was during asssigning colors to the arrow keys .I initially don't know where to start and later I used *myscreen.onkey function and tried multiple times to get it.
```

Comment:
Quick typo with the period after keys--looks like there might be an extra space!

Reply:
Thanks for the suggestions haley. I have corrected the typos and submitted it . can you review it again when you got free time.Thank you:)